app/frontend/src/internationalization/trans/chefs/hi/hi.json

```diff
@@ -662,40 +660,40 @@
     "notSaved": "सहेजा नहीं गया"
   },
   "formViewer": {
-    "lateFormSubmissions": "प्रपत्र जमा करने की अवधि समाप्त हो गई है! आप अभी भी नीचे दिए गए बटन पर क्लिक करके देर से सबमिशन बना सकते हैं।",
+    "lateFormSubmissions": "फॉर्म जमा करने की अवधि समाप्त हो गई है! आप अभी भी नीचे दिए गए बटन पर क्लिक करके देर से सबमिशन बना सकते हैं।",
     "createLateSubmission": "देर से सबमिशन बनाएं",
-    "draftSaved": "ड्राफ़्ट सहेजा गया",
+    "draftSaved": "ड्राफ्ट सहेजा गया",
     "saving": "सहेजा जा रहा है",
     "pleaseConfirm": "कृपया पुष्टि करें",
-    "submitFormWarningMsg": "क्या आप वाकई अपना फ़ॉर्म सबमिट करना चाहते हैं?",
+    "submitFormWarningMsg": "क्या आप वाकई अपना फॉर्म जमा करना चाहते हैं?",
     "submit": "जमा करना",
-    "wantToSaveDraft": "क्या आप ड्राफ़्ट सहेजना चाहते हैं?",
-    "version": "संस्करण: {version}",
-    "formScheduleExpireMessage": "फॉर्म सबमिशन उपलब्ध नहीं है क्योंकि निर्धारित सबमिशन अवधि समाप्त हो गई है।",
-    "getUsersSubmissionsErrMsg": "इस फॉर्म के लिए सबमिशन लाने में एक त्रुटि हुई",
-    "getUsersSubmissionsConsoleErrMsg": "फ़ॉर्म सबमिशन डेटा लोड करने में त्रुटि {submissionId}: {error}",
-    "multiDraftUploadSuccess": "आपका एकाधिक ड्राफ़्ट अपलोड सफल रहा है!",
-    "readVersionErrMsg": "प्रतिक्रिया में कोई स्कीमा नहीं। संस्करण आईडी: {versionId}",
-    "readDraftErrMsg": "प्रतिक्रिया में कोई स्कीमा नहीं। ड्राफ्टआईडी: {ड्राफ्टआईडी}",
-    "alertRouteMsg": "प्रपत्र स्वामी ने प्रपत्र प्रकाशित नहीं किया है, और यह सबमिशन के लिए उपलब्ध नहीं है।",
-    "fecthingFormErrMsg": "इस फॉर्म को लाने में एक त्रुटि हुई",
+    "wantToSaveDraft": "क्या आप ड्राफ्ट सहेजना चाहते हैं?",
+    "version": "संस्करण: {संस्करण}",
+    "formScheduleExpireMessage": "फॉर्म जमा करना उपलब्ध नहीं है क्योंकि जमा करने की निर्धारित अवधि समाप्त हो गई है।",
+    "getUsersSubmissionsErrMsg": "इस फ़ॉर्म के लिए सबमिशन लाते समय एक त्रुटि उत्पन्न हुई",
+    "getUsersSubmissionsConsoleErrMsg": "फ़ॉर्म सबमिशन डेटा लोड करने में त्रुटि {submissionId}: {त्रुटि}",
+    "multiDraftUploadSuccess": "आपका एकाधिक ड्राफ्ट अपलोड सफल रहा है!",
+    "readVersionErrMsg": "प्रत्युत्तर में कोई स्कीमा नहीं. संस्करणआईडी: {संस्करणआईडी}",
+    "readDraftErrMsg": "प्रत्युत्तर में कोई स्कीमा नहीं. ड्राफ्टआईडी: {ड्राफ्टआईडी}",
```

Comment:
This parameter was translated in the previous version of the file, I think it should be `{draftId}`